packages/patternfly-4/react-core/src/components/Nav/NavGroup.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Nav/styles.css';
+import a11yStyles from '@patternfly/patternfly-next/utilities/Accessibility/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import NavToggle from './NavToggle';
+import { AngleRightIcon } from '@patternfly/react-icons';
+
+const propTypes = {
+  id: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  ariaLabel: PropTypes.string.isRequired,
```

Comment:
should be aria-label.
I have the same question here about the prop not being required.  Should we default to something meaningful, or generate a warning?

Reply:
I think we should generate a warning here for aria-label

Reply:
I actually think that  ariaLabel can be removed here . In the design it looks like it is only using aria-labelledby .